kotsadm/operator/pkg/client/client.go

```diff
@@ -85,7 +86,7 @@ func (c *Client) Run() error {
 
 	supportbundle.StartServer()
 
-	c.runHooksInformer()
+	c.runHooksInformer(c.TargetNamespace)
```

Comment:
How about adding `TargetNamespace` to `ExistingInformers` as well for consistency.

Reply:
done